contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -75,7 +83,9 @@
 
   private Row currentRow;
 
-  private Workbook workbook;
+  private StreamingWorkbook swb;
```

Comment:
Is there any reason for using specific implementation instead of the interface here?

Reply:
@vvysotskyi 
The way the underlying library is written, in order to access the metadata, we need the workbook to be a `StreamingWorkbook` type, but for whatever reason, the reader only returns the `Workbook` type.

Reply:
This class uses only methods from the `Workbook` interface, so I don't see any reason for narrowing down the type to `StreamingWorkbook` here and adding explicit casts for this type.

Reply:
The `getCoreProperties()` unfortunately is not in the `Workbook` class.  I'm remembering now that was why I had to make that cast.